src/ray/raylet/main.cc

```diff
@@ -32,6 +32,8 @@ DEFINE_int32(node_manager_port, -1, "The port of node manager.");
 DEFINE_int32(metrics_agent_port, -1, "The port of metrics agent.");
 DEFINE_int32(metrics_export_port, 1, "Maximum startup concurrency");
 DEFINE_string(node_ip_address, "", "The ip address of this node.");
+DEFINE_string(gcs_address, "", "The IP address of GCS server.");
+DEFINE_int32(gcs_port, -1, "The port of GCS server.");
```

Comment:
Question: I thought we decided to include port in gcs_address

Reply:
Yes that is my initial thought too. But I see that raylet already uses separate flags for ip and port. GcsClientOptions also has separate fields for them. So using separate flags for GCS ip and port might be easier. Up to you to decide.